modules/statics/src/coins.ts

```diff
@@ -53,6 +53,8 @@ export const coins = CoinMap.fromCoins([
     ALGO_FEATURES,
     KeyCurve.Ed25519
   ),
+  account('avaxc', 'Avalanche C-Chain', Networks.main.avalancheC, 18, UnderlyingAsset.AVAXC, ETH_FEATURES),
+  account('tavaxc', 'Testnet Avalanche C-Chain', Networks.main.avalancheC, 18, UnderlyingAsset.AVAXC, ETH_FEATURES),
```

Comment:
Shouldn't the network be `avalanchCTestnet` here?

Reply:
it should be `Networks.test.avalancheC`
Fixed